.travis.yml

```diff
@@ -3,15 +3,23 @@ language: node_js
 node_js:
   - "5.5.0"
 
+cache:
+  directories:
+    - node_modules
+
+addons:
+  firefox: latest-beta
+
 before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
+  - npm config set spin false
+  - npm install
+
 before_script:
-  - cd $TRAVIS_BUILD_DIR
-  - ./node_modules/.bin/mozilla-download --branch mozilla-central --product firefox firefox
```

Comment:
Nit: remove unused **mozilla-download** from the [package.json](https://github.com/mozilla/activity-streams/blob/f7eaf012810a5dc7c61bbce10c475aeb29556225/package.json#L71) file.